Server/test/controllers/AuthenticatorControllerSpec.scala

```diff
@@ -0,0 +1,28 @@
+package controllers
+
+import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.guice.GuiceOneAppPerTest
+import play.api.mvc.Result
+import play.api.test.Helpers.contentAsString
+import play.api.test.{FakeRequest, Helpers, Injecting}
+
+import scala.concurrent.Future
+
+class AuthenticatorControllerSpec extends PlaySpec
```

Comment:
Can you comment out this class, it's causing the build to fail.

Reply:
Done